astroquery/alma/__init__.py

```diff
@@ -38,8 +38,8 @@ class Conf(_config.ConfigNamespace):
 
 conf = Conf()
 
-from .core import Alma, AlmaClass, ALMA_BANDS
+from .core import Alma, AlmaClass, ALMA_BANDS, get_enhanced_table
 
 __all__ = ['Alma', 'AlmaClass',
-           'Conf', 'conf', 'ALMA_BANDS'
+           'Conf', 'conf', 'ALMA_BANDS', 'get_enhanced_table'
```

Comment:
I wonder if this would/will be useful for any of the other modules, too?

If yes, then it may need to live in another, more generic namespace instead.

Reply:
That would be a good idea except that this `s_region` parser is not generic enough except for ALMA and it has been decided that STC-S is a dead end and it's not worth writing a generic one.

Reply:
ok, thanks for the explanation